src/test/clojure/jarl/builtins/time_test.clj

```diff
@@ -20,6 +21,37 @@
       ; exceptions
       [9655283296943749000]      [BuiltinException "timestamp too big"])))
 
+(deftest builtin-time-parse-duration-ns-test
+  (testing-builtin "time.parse_duration_ns"
+    ["1ns"]                     1
+    ["1ns2ms"]                  2000001
+    ["1h"]                      3600000000000
+    ["33h11m59s7ms99us12345ns"] 119519007111345
+    ; exceptions
+    ["h"]                       [BuiltinException "time: invalid duration \"h\""]
+    ["33"]                      [BuiltinException "time: missing unit in duration \"33\""]
+    ["33ks"]                    [BuiltinException "time: unknown unit \"ks\" in duration \"33ks\""]))
+
+(deftest builtin-time-parse-ns-test
```

Comment:
Hmm, that's not good! What version of Java are you using there? I wonder if we can have that fixed for the project somehow 🤔 It seems a lot of stuff like this differs between versions..

To start with I guess it would be good to have a lein task that simply printed the JVM version in use.. that'd be great to compare potential differences between machines, and the CI environment, in the future.

Reply:
I've done more work on this in a follow-up PR, so I agree that we should get this merged either way.. let's see if you still see the same issue once that's up.

Reply:
Yea, I agree that the most likely culprit here might be the JVM. I'm running 17.0.1 from Oracle.